helm/observability-bundle/values.yaml

```diff
@@ -115,3 +115,6 @@ apps:
       - kind: configMap
         name: "{{ $.Values.clusterID }}-grafana-agent-config"
         namespace: "{{ $.Release.Namespace }}"
+      - kind: secret
+        name: "{{ $.Values.clusterID }}-grafana-agent-secret"
+        namespace: kube-system
```

Comment:
Why kube-system?

Reply:
Because I need this secret where the agent is running to use `envFrom`

Reply:
So { $.Release.Namespace }} right?

Reply:
Humm it's not what I see.
{ $.Release.Namespace }} => namespace where the app is deployed, so `giantswarm`

Reply:
Well yes but then that will never work on WCs :)

Reply:
Beause this secret would be created on the MC cluster